src/Couchbase.Lite.Shared/API/Query/Parameters.cs

```diff
@@ -130,8 +126,7 @@ public Parameters SetBoolean([NotNull]string name, bool value)
         /// <param name="name">The name of the key to set</param>
         /// <param name="value">The value to set</param>
         /// <returns>The parameters object for further processing</returns>
-        [NotNull]
-        public Parameters SetDate([NotNull]string name, [CanBeNull]DateTimeOffset value)
+        public Parameters SetDate(string name, DateTimeOffset value)
```

Comment:
DateTimeOffset is CanBeNull.

Reply:
DateTimeOffset cannot be null, this attribute was wrong.  It is a struct, so the compiler does not allow null.